modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -5,41 +5,53 @@
 [id="op-constructing-pipelines-using-pipeline-builder_{context}"]
 = Constructing Pipelines using the Pipeline Builder
 
-In the *Developer* perspective of the console, you can use the *Add* -> *Pipeline* -> *Pipeline Builder* option to:
+In the *Developer* perspective of the console, you can use the *+Add* -> *Pipeline* -> *Pipeline Builder* option to:
 
-* Construct a Pipeline flow using existing Tasks and ClusterTasks. When you install the OpenShift Pipelines Operator, it adds reusable Pipeline ClusterTasks to your cluster.
-* Specify the type of resources required for the Pipeline Run, and if required, add additional parameters to the Pipeline.
-* Reference these Pipeline resources in each of the Tasks in the Pipeline as input and output resources.
-* The parameters for a Task are prepopulated based on the specifications of the Task. If required, reference any additional parameters added to the Pipeline in the Task.
+* Configure pipelines using either the *Pipeline builder* or the *YAML view*.
```

Comment:
In the UI it's _Pipeline Builder_, and IBM says to always use the UI convention.

Reply:
It is "Pipeline builder" only in the UI also. Please check the screenshot attached:
![image](https://user-images.githubusercontent.com/3268242/107514127-3ebe6700-6bcf-11eb-9a5b-7b9aa05e9bed.png)

Reply:
Oh, okay, it looked different in a different screen capture.